apps/condo/domains/meter/components/MeterReportingPeriodForm.tsx

```
@@ -379,67 +383,67 @@ export const MeterReportingPeriodForm: React.FC<IMeterReportingPeriodForm> = ({
+                                                actions={[
+                                                    <ButtonWithDisabledTooltip
+                                                        key='submit'
+                                                        type='primary'
+                                                        disabled={isDisabled}
+                                                        title={errors}
+                                                        onClick={handleSave}
+                                                        loading={isLoading}
+                                                    >
+                                                        {isCreateMode ? SubmitButtonCreateLabel : SubmitButtonApplyLabel}
+                                                    </ButtonWithDisabledTooltip>,
+                                                    isCreateMode ? <></> : <DeleteButtonWithConfirmModal
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)